README.md

```diff
@@ -1053,6 +1053,22 @@ Other Style Guides
     };
     ```
 
+  <a name="whitespace--implicit-arrow-linebreak"></a><a name="8.6"></a>
```

Comment:
thus, numbered links are never to be added - only preserved forever on legacy items.